resources/assets/less/app/kanban.less

```diff
@@ -54,6 +54,110 @@
   }
 }
 
+
+/* navpill */
+.kanban-nav {
+	position: relative;
+	margin: 0px 0;
+}
+.kanban-pill-tag {
+	color: #afafaf;
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+.kanban-nav ul {
+	margin: 0;
+	padding: 0;
+}
+.kanban-nav li {
+	margin: 0 5px 10px 0;
+	padding: 0;
+	list-style: none;
+	display: inline-block;
+}
+.kanban-nav a {
```

Comment:
Found out the tabs where in the original snippet. That's why they're accidentally there. I'll remove them.